### PR TITLE
Tejashree_Assignment_5

### PR DESCRIPTION
**Pull request by TejashreeTOct2020**
Added for practice

### CODE REVIEW

**Review by harshvegada — COMMENTED** [review]
please add assignment 5 questions.

**Comment by TejashreeTOct2020** [comment]
added codes for assignment 5. Please review.

File: src/tejashree/Mall.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		paymentChoice = byVisa;
+	}
+	void paymentDoneUsingCash(String byCash) {
+		paymentChoice = byCash;
+	}
+	void displayCustomerInfo() {
+		System.out.println(custName + " added " + itemCount + " items and payment done using " + paymentChoice + section);
+	}
+	void needBill(boolean needBill) {
+		if (!needBill) {
```
Comment (harshvegada) [inline src/tejashree/Mall.java:26]: why do we need ! (not) condition here,

File: src/tejashree/Mall.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+			System.out.println(custName + " needs a copy of bill");
+		} else
+			System.out.println(custName + " does not need a bill copy");
+	}
+	void haveCarryBag(boolean needBag) {
+		if (!needBag) {
```
Comment (harshvegada) [inline src/tejashree/Mall.java:32]: we are changing the value internally which is not requirement.

File: src/tejashree/Mall.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+	}
+	public static void main(String[] args) {
+		Mall mall = new Mall();
+		mall.customerName("Tejashree");
+		mall.itemAddedToCart(9);
+		mall.paymentDoneUsingCash("cash");
+		mall.decideSection();
+		mall.haveCarryBag(true);
+		mall.displayCustomerInfo();
```
Comment (harshvegada) [inline src/tejashree/Mall.java:56]: needBill() method not get called, please correct it

**Review by harshvegada — COMMENTED** [review]
Hi @TejashreeTOct2020 
Please update code, & add missing method call()

**Comment by TejashreeTOct2020** [comment]
@harshvegada i have updated as per your comments.

**Pull request merged**